mintpy/objects/gps.py

```diff
@@ -16,7 +16,7 @@
 from urllib.request import urlretrieve
 
 from mintpy.objects.coord import coordinate
-from mintpy.utils import ptime, time_func, readfile, utils1 as ut
+from mintpy.utils import ptime, time_func, readfile, utils1 as ut, utils as ut0
```

Comment:
`utils1` already has everything from `utils0`, so this additional import is not needed.
https://github.com/insarlab/MintPy/blob/670c6a9f9657f883ad6563d4f4e66bf6f8c7aac3/mintpy/utils/utils1.py#L21

Reply:
'coordinate' module used directly now.